### PR TITLE
Grid + Reviews Components

### PR DESCRIPTION
**Pull request by tejas-muthya37**

### CODE REVIEW

File: Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            </pre>
+          </div>
+
+          <h1 class="sub-heading">Simple Grid - 3 columns</h1>
+
+          <p>A Simple Grid with 3 columns in a row.</p>
+
+          <div class="demo-with-code">
+            <div class="demo-div">
+              <div class="grid grid-3">
```
Comment (SJTGSHIVAM) [inline Grid/grid.html:101]: Looks clean

File: Grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.grid {
+    display: grid;
+    width: 100%;
+}
+
```
Comment (SJTGSHIVAM) [inline Grid/grid.css:5]: names are picked up aptly that is good for picking them up

**Review by SJTGSHIVAM — APPROVED** [review]
I like the way you added a small amount of code in pr. things look just fine.

**Review by ivinayakg — APPROVED** [review]
The most important thing about any PR is that the code and the title should connect and also the code should be self-explanatory and your code does both, Great Work 👍

**Pull request merged**